cardano-crypto-class/src/Cardano/Crypto/KES/Class.hs

```diff
@@ -261,3 +266,37 @@ encodeSignedKES (SignedKES s) = encodeSigKES s
 
 decodeSignedKES :: KESAlgorithm v => Decoder s (SignedKES v a)
 decodeSignedKES = SignedKES <$> decodeSigKES
+
+--
+-- 'Size' expressions for 'ToCBOR' instances.
+--
+
+-- | 'Size' expression for 'VerKeyKES' which is using 'sizeVerKeyKES' encoded
+-- as 'Size'.
+--
+encodedVerKeyKESSizeExpr :: forall v. KESAlgorithm v => Proxy (VerKeyKES v) -> Size
+encodedVerKeyKESSizeExpr _proxy =
+      -- 'encodeBytes' envelope
+      fromIntegral ((withWordSize :: Word -> Integer) (sizeVerKeyKES (Proxy :: Proxy v)))
+      -- payload
+    + fromIntegral (sizeVerKeyKES (Proxy :: Proxy v))
```

Comment:
These could also use the ByteString instance with the fixed size, like the Hash instance does.

Reply:
Not every instance of `VerKeyKES` is wrapping a `ByteString`. Even though you would get a correct result, I don't think this is right.